styles/colors-silicon-night.less

```diff
@@ -7,6 +7,13 @@
 // as these are overrides to the "base" layer of colors.
 //
 
+// Abstracted Colors
+@agate-background: black;
+@agate-foreground: white;
+@agate-background-active: fade(white, 15%);
+@agate-foreground-active: black;
+@agate-foreground-focus: black;
```

Comment:
Let's refer to the `@agate` or `@silicon` color variables here instead of the hard-coded color-words.